Block/Adminhtml/Order/View.php

```diff
@@ -177,9 +180,9 @@ public function getReadCheckoutAPI()
         $checkout_status = $responseBody['checkout_status'] ?: null;
 
         // Read charge endpoint if charge id exists
-        $charge_id = $this->getChargeId();
-        if ($charge_id) {
-            $readChargeResponse = $this->affirmCheckout->readCharge($charge_id);
+        $transaction_id = $this->getChargeId();
```

Comment:
Should we call it `getTransactionId` and update the docs and method names as well.